Automattic/wp-calypso
# Simple payments block: fix 404ing image due @ character

When building Jetpack blocks using the SDK, Webpack+wpcom CDN didn't play nicely with `@` character, thus using `-` is safer.

SDK creates files with `%40` symbol and keeps URLs with `@` symbol. While that works fine with some environments (e.g. in self-hosted Ubuntu/Apache wp-admin), accessing the file path via WordPress.com CDN gives _"404 not found"_. 

I'm not sure if the issue is with the SDK/Webpack or with the CDN.

The URL:
```
paypal-button@2x-fe4d34770a47484f401cecbb892f8456.png
```

Actual filename:
```
paypal-button%402x-fe4d34770a47484f401cecbb892f8456.png
```

This won't fix whatever is the underlying issue but unblocks us for now.

Fixes #29053


### Testing instructions

- Build blocks to your sandbox locally and sync the files up to your sandbox:
   ```
    npm run sdk -- gutenberg client/gutenberg/extensions/presets/jetpack/ \
        --output-dir=YOUR_SANDBOX_PATH/wp-content/mu-plugins/jetpack/_inc/blocks-staging/ \
        --watch
    ```
- Have a site with Premium+ plan
- Insert Simple payments block in Gutenberg
- On hi-density screen, observe 2x image load properly:
    ```
    /wp-content/mu-plugins/jetpack/_inc/blocks-staging/images/paypal-button-2x-fe4d34770a47484f401cecbb892f8456.png
    ```
    
<img width="217" alt="image" src="https://user-images.githubusercontent.com/87168/49446702-79513b80-f7dd-11e8-8f00-9f94262663a4.png">
